test/uk/gov/hmrc/cdsreimbursementclaimfrontend/controllers/claims/EnterDuplicateMovementReferenceNumberSpec.scala

```diff
@@ -147,6 +150,11 @@ class EnterDuplicateMovementReferenceNumberSpec
         val result          = controller.enterDuplicateMrnSubmit()(
           FakeRequest().withFormUrlEncodedBody("enter-movement-reference-number" -> "123456789A12345678")
         )
+
```

Comment:
Try using this method everywhere:

```
        checkPageIsDisplayed(
          performAction(
            Seq.empty
          ),
          messageFromMessageKey("select-who-is-making-the-claim.title"),
          doc =>
            doc
              .select(".govuk-error-summary__list > li > a")
              .text() shouldBe messageFromMessageKey(
              s"select-who-is-making-the-claim.error.required"
            ),
          BAD_REQUEST
        )
```

Reply:
Why is that better than what I have?  I copied it from BankAccountControllerSpec - don't they do the same thing?